src/main/java/org/openmainframeproject/tersedecompress/TerseDecompress.java

```diff
@@ -95,18 +103,49 @@ else if (outputFileName == null)
     		printUsageAndExit();
     	}
 
+		System.out.println("Attempting to decompress input file (" + inputFileName +") to output file (" + outputFileName +")");
```

Comment:
Here is an example how it will be more streamlined: 
```suggestion
    System.out.println("Opening output file (" + outputFileName + ")");
    var outputStream = null;
    try {
      if (outputFileName.endsWith(".gz")) {
        outputStream = new GZIPOutputStream(new FileOutputStream(outputFileName), 8192, true);
      }
      else {
        outputStream = new FileOutputStream(outputFileName);
      }
    }
    catch (Exception e) {
      System.out.println("Got exception while opening output file (" + outputFileName + ").\nError message:\n" + e.toString());
    }
    System.out.println("Attempting to decompress input file (" + inputFileName + ") to output file (" + outputFileName + ")");
    try (TerseDecompresser outputWriter = TerseDecompresser.create(new FileInputStream(inputFileName), outputStream)) {	 
      outputWriter.TextFlag = textMode;
      outputWriter.decode();
    }
    catch (Exception e) {
      System.out.println("Got exception while decompressing input file (" + inputFileName + ").\nError message:\n" + e.toString());
    }
```

Reply:
Updated